packages/cypress/src/support.ts

```diff
@@ -32,7 +32,8 @@ afterEach(() => {
         cy.task('prepareArchives', {
           action: 'save-archives',
           payload: {
-            testTitle: Cypress.currentTest.title,
+            // @ts-expect-error relativeToCommonRoot is on spec (but undocumented)
+            testTitlePath: [Cypress.spec.relativeToCommonRoot, ...Cypress.currentTest.titlePath],
```

Comment:
This isn't ideal, as `relativeToCommonRoot` is undocumented. I didn't see another way of figuring out the path from the tests directory to the tests file itself. `Cypress.spec.absolute` gives the absolute path to the test file, but it takes some deducing to know how to get from the file to the tests folder root.